.github/workflows/ci.yml

```diff
@@ -113,7 +113,18 @@ jobs:
       matrix:
         python-version: ["3.7", "3.8", "3.9", "3.10"]
         db-backend: ["postgresql"]
-        nautobot-version: ["1.3.3", "stable"]
+        nautobot-version: ["stable"]
```

Comment:
Feel like this is unneeded based on above matrix

Reply:
This was used to limit the matrix, not expand

Reply:
added comment per @whitej6